pkg/ebpf/c/types.h

```diff
@@ -307,8 +307,14 @@ typedef struct config_entry {
     u64 pid_min;
 } config_entry_t;
 
+enum capture_options_e
+{
+    NET_CAP_OPT_FILTERED = (1 << 0), // pcap should obey event filters
```

Comment:
I might enter NET_CAP_OPT_SCOPED here, for example. And maybe other future options.